dali/test/python/test_operator_get_property.py

```diff
@@ -52,29 +52,43 @@ def test_file_properties():
 
 
 @pipeline_def
-def wds_properties(root_path, device):
-    read = fn.readers.webdataset(paths=[root_path], ext=['jpg'])
+def wds_properties(root_path, device, idx_paths):
+    read = fn.readers.webdataset(paths=[root_path], index_paths=idx_paths, ext=['jpg'])
     if device == 'gpu':
         read = read.gpu()
     return fn.get_property(read, key="source_info")
 
 
-def _test_wds_properties(device):
+def generate_wds_index(root_path, index_path):
+    from wds2idx import IndexCreator
+    ic = IndexCreator(root_path, index_path)
```

Comment:
In the documentation of the tool you use a different syntax (`with ... as ic:`)

Reply:
Nice catch. Fixed.